src/main/java/com/mparticle/kits/TaplyticsKit.java

```diff
@@ -165,6 +162,57 @@ public void removeUserAttribute(String attribute) {
         setUserAttribute(attribute, null);
     }
 
+    @Override
+    public void onIdentifyCompleted(MParticleUser mParticleUser, FilteredIdentityApiRequest filteredIdentityApiRequest) {
+        setUserAttributeFromRequest(filteredIdentityApiRequest);
+    }
+
+    /**
+     * Identity Listener
+     */
+
+    public void setUserAttributeFromRequest(FilteredIdentityApiRequest request) {
+        Map<MParticle.IdentityType, String> identities = request.userIdentities;
+        try {
+            JSONObject attr = new JSONObject();
+            if (identities.get(IdentityType.CustomerId) != null) {
+                attr.put(USER_ID, identities.get(IdentityType.CustomerId));
+            }
+            if (identities.get(IdentityType.Email) != null) {
+                attr.put(EMAIL, identities.get(IdentityType.Email));
+            }
+            Taplytics.setUserAttributes(attr);
+        } catch (JSONException e) {
+
+        }
+    }
+
+    @Override
+    public void onLoginCompleted(MParticleUser user, FilteredIdentityApiRequest request) {
+        setUserAttributeFromRequest(request);
+    }
+
+    @Override
+    public void onLogoutCompleted(MParticleUser mParticleUser, FilteredIdentityApiRequest filteredIdentityApiRequest) {
+        Taplytics.resetAppUser(new TaplyticsResetUserListener() {
+            @Override
+            public void finishedResettingUser() {
+                // no-op
+            }
+        });
+    }
+
+    @Override
+    public void onModifyCompleted(MParticleUser mParticleUser, FilteredIdentityApiRequest filteredIdentityApiRequest) {
+        // no-op
```

Comment:
Hey @jsalaber I want to point out that the state of the current user's `IdentityType`s can change on `modify()` (as well as `logout()`. Did you mean to logic in place that they will only be updated in Taplytics for certain Identity requests or should they change _whenever_ the Identity value changes?

Reply:
hey @willpassidomo, what kind of things can change in `onModifyCompleted`, can customer id change?

Reply:
i was just mirroring the iOS integration's functionality, but was wondering if I also need to implement `onModifyCompleted` in ios then

Reply:
@jsalaber Yes, customerId can change on modify(). Probably not be a common use-case (honestly I don't have visibility into that kind of thing) but as far as our SDK logic goes, it would be totally valid

Reply:
hey @willpassidomo i rebased and added some code to `onModify`, let me know when you can take a look!